### PR TITLE
Feature/quiz and cms

### PR DESCRIPTION
**Pull request by MohdAnas**
I have done some changes in the code please review it thoroughly. Thanks

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -15,6 +15,10 @@ class Application < Rails::Application
     config.load_defaults 5.2
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{rb,yml}')]
```
Comment (MohdAnas) [inline config/application.rb:17]: I found that this is the API only app, so I have to add these middlewares to support cookies and flash.

**Comment by MohdAnas** [comment]
> Great work, looks awesome!
> Few possible amendments -
> i. We can make the fields more strict in the schema (by adding `null: false`, especially for the _boolean_ fields) to prevent any kind of goofup later.
> ii. Personally I would prefer to remove the boolean `correct` field from the `answers` table, knowing that it would be false 75% of the time and wont generally get updated.
> Something on the line of this -
> a. Drop `correct` boolean field from `answers` table, also rename this table to `options` table.
> b. Create a new table `answers` with `question_id` and `option_id`, it will store the correct option for every question.
> Just an alternative way. Maybe we can wait for @nronas inputs here.

**Comment by MohdAnas** [comment]
Thanks for review. I will update it for boolean columns(null: false). Thanks

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,20 @@
...
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_04_140723) do
+ActiveRecord::Schema.define(version: 2020_05_10_184356) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "answers", force: :cascade do |t|
+    t.text "answer_text"
```
Comment (nronas) [inline db/schema.rb:19]: since we are already in the answers table, lets rename this as:

```suggestion
    t.text "name"
```

File: db/schema.rb [hunk 3]
```diff
@@ -10,11 +10,20 @@
...
+    t.bigint "question_id"
```
Comment (nronas) [inline db/schema.rb:20]: 💅 
```suggestion
    t.references :question
```

File: db/schema.rb [hunk 4]
```diff
@@ -10,11 +10,20 @@
...
+    t.boolean "correct", default: false, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```
Comment (nronas) [inline db/schema.rb:23]:
```suggestion
    t.timestamps
```

File: db/schema.rb [hunk 5]
```diff
@@ -10,11 +10,20 @@
...
+    t.index ["question_id"], name: "index_answers_on_question_id"
```
Comment (nronas) [inline db/schema.rb:24]: if we do https://github.com/solocoinapp/solocoin-backend/pull/69/files#r422975083 we shouldn't need this

File: db/schema.rb [hunk 6]
```diff
@@ -61,6 +70,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "questions", force: :cascade do |t|
+    t.text "question_text"
```
Comment (nronas) [inline db/schema.rb:74]: Same as above:

```suggestion
    t.text "name"
```

File: db/schema.rb [hunk 7]
```diff
@@ -61,6 +70,13 @@
...
+    t.boolean "active", default: true, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```
Comment (nronas) [inline db/schema.rb:77]:
```suggestion
    t.timestamps
```

File: db/schema.rb (hunk 2 above)
Comment (nronas) [reply db/schema.rb:19]: lol i just found out that this is the schema :D :D :D

File: db/migrate/20200510131345_create_questions.rb [hunk 8]
```diff
@@ -0,0 +1,10 @@
+class CreateQuestions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :questions do |t|
+      t.text :question_text
```
Comment (nronas) [inline db/migrate/20200510131345_create_questions.rb:4]:
```suggestion
      t.string :name
```

File: db/migrate/20200510144711_create_answers.rb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :answers do |t|
+      t.text :answer_text
```
Comment (nronas) [inline db/migrate/20200510144711_create_answers.rb:4]:
```suggestion
      t.string :name
```

File: db/schema.rb (hunk 2 above)
Comment (MohdAnas) [reply db/schema.rb:19]: Yes I am just about to say that, LOL

**Comment by MohdAnas** [comment]
> Some suggestions but this LGTM

Thanks for the review.

**Pull request merged**